apps/site/src/components/modal/create-schema-modal.tsx

```diff
@@ -69,8 +69,7 @@ export const CreateSchemaModal: FunctionComponent<CreateSchemaModalProps> = ({
       if (error) {
         setApiErrorMessage(error.message);
       } else if (data) {
```

Comment:
(Comment for line 39)
We allow spaces in the schema titles (we do in HASH). Should the types here behave different in that regard?

Reply:
Good flag – updated in https://github.com/blockprotocol/blockprotocol/pull/981/commits/5f8ac4a852469707588c10bf083b699bcd5de3b3 to just trim and remove most special characters (we can add them back in by request)